tRecorderApi/api/tests_helpers.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+from views_sets import md5Hash
+
+class HelperTestCases(TestCase):
+
+    def unit_testing_md5Hash_method_for_equality_with_duplicate_wav_files(self):
+        # create object later when this is added to views_sets.py
+        # helpers.md5Hash('chapter.wav')
+        # helpers.md5Hash('chapter.wav')
+        self.assertEqual(md5Hash('chapter.wav'), md5Hash('chapter.wav'))
+
+
+    def unit_testing_md5Hash_method_for_inequality_with_different_wav_files(
```

Comment:
This looks like it is testing that  a python lib does its job.  Is that the case?